src/Sulu/Bundle/AdminBundle/Resources/js/services/Router/Router.js

```diff
@@ -113,19 +118,29 @@ export default class Router {
     }
 
     @action update(name: string, attributes: Object) {
-        this.route = routeRegistry.get(name);
+        const route = routeRegistry.get(name);
+
+        const updatedAttributes = {
+            ...this.updateAttributesHooks.reduce((hookAttributes, updateAttributeHook) => ({
```

Comment:
That does not work because the attribute hook might return `undefined` for some attributes, and then `undefined` would override also the defaults from before.